common/djangoapps/student/models.py

```diff
@@ -2561,12 +2626,13 @@ def __unicode__(self):
 
     @classmethod
     def add_enrollment_attr(cls, enrollment, data_list):
-        """Delete all the enrollment attributes for the given enrollment and
+        """
+        Delete all the enrollment attributes for the given enrollment and
         add new attributes.
 
         Args:
             enrollment(CourseEnrollment): 'CourseEnrollment' for which attribute is to be added
```

Comment:
Can you put a space between `enrollment` and `(CourseEnrollment)` while you're here?